demo/c/cobra_demo_file.c

```diff
@@ -233,18 +262,21 @@ int main(int argc, char *argv[]) {
 #define NULL_TERMINATED (-1)
 
     LPWSTR *wargv = CommandLineToArgvW(GetCommandLineW(), &argc);
-    if (wargv == NULL) {
+    if (wargv == NULL)
+    {
```

Comment:
Why? We don't use this format of curly brace for any other functions

Reply:
because that's a mistake :) I am not sure why clion didn't fix it!

Reply:
got it! because of #if directives, those parts of code seem invisible to clion